### PR TITLE
dom-event-delegation - Lamberto Martinez

### PR DESCRIPTION
**Pull request by Lambpato**
https://user-images.githubusercontent.com/34325124/217896923-6c6c1ab6-f46c-4deb-b770-ee22aeee86c7.mov

### CODE REVIEW

File: dom-event-delegation/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var $taskList = document.querySelector('.task-list');
+function test(e) {
```
Comment (shawnkost) [inline dom-event-delegation/main.js:2]: You should not be naming anything `test`. It's not descriptive of what the function actually does

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested here Lamberto

File: dom-event-delegation/main.js (hunk 1 above)
Comment (Lambpato) [reply dom-event-delegation/main.js:2]: my bad i forgot to change it after i was done testing out the first step of the exercise

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**